### PR TITLE
added timer for plots

### PR DESCRIPTION
**Pull request by msvinaykumar**
Timer for box plots added 

Sample output
# HELP KruizeMethod_max Time taken for Kruize methods
# TYPE KruizeMethod_max gauge
KruizeMethod_max{application="Kruize",method="generatePlots",status="success",} 0.036112854
# HELP KruizeMethod Time taken for Kruize methods
# TYPE KruizeMethod summary
KruizeMethod_count{application="Kruize",method="generatePlots",status="success",} 2.0
KruizeMethod_sum{application="Kruize",method="generatePlots",status="success",} 0.050705769

### CODE REVIEW

**Review by khansaad — APPROVED** [review]
LGTM!

File: design/KruizePromQL.md [hunk 1]
```diff
@@ -81,29 +93,39 @@ The following are the available Kruize DB methods that you can monitor:
 - `addPerformanceProfileToDB`: Method to add performance profile to the database.
 - `loadPerformanceProfileByName`: Method to load a specific performance profile.
 - `loadAllPerformanceProfiles`: Method to load all performance profiles.
+- `genratePlots`: methode to generate plots for all terms.
```
Comment (dinogun) [inline design/KruizePromQL.md:96]: It should be `generatePlots`. `method` is wrongly spelled as well

Comment (dinogun) [reply design/KruizePromQL.md:96]: Please change it to: `Method to generate blox plot metrics for all terms`

Comment (msvinaykumar) [reply design/KruizePromQL.md]: done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**